src/providers/wfs/qgswfsdataitems.h

```diff
@@ -79,6 +79,23 @@ class QgsWfsLayerItem : public QgsLayerItem
     QgsWfsLayerItem( QgsDataItem *parent, QString name, const QgsDataSourceUri &uri, QString featureType, QString title, QString crsString );
     ~QgsWfsLayerItem();
 
+    virtual QList<QMenu *> menus() override;
+
+  public slots:
+    //! get style of the active data item (geonode layer item) and copy it to the clipboard
+
+    /**
+       \param sourceItem  The data item where the style will be taken from
```

Comment:
I cannot find this param in the signature. ... and the comment looks a bit wierd to me: I would use `/**` and merge the line 85 in.